# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../metamail/85_mm-2.7-rm.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# Copyright (C) 1998 - 2003 ROCK Linux Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

diff -Nur mm2.7.orig/src/Makefile mm2.7/src/Makefile
--- mm2.7.orig/src/Makefile	2003-05-15 00:19:21.000000000 +0200
+++ mm2.7/src/Makefile	2003-05-15 00:21:59.000000000 +0200
@@ -15,7 +15,7 @@
 # System V users may change the following to remove the "-s" and all will still work.
 LN = ln -s
 
-RM = rm
+RM = rm -f
 # To shut rm up on SGI machines (where symlinks have mode 0000)
 # RM = rm -f
 
